amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpConnectorSettings.scala

```diff
@@ -135,18 +135,41 @@ object TemporaryQueueSourceSettings {
     TemporaryQueueSourceSettings(connectionProvider, exchange)
 }
 
+final class AmqpPublishConfirmConfiguration private (
```

Comment:
Does the confirm timeout require a configuration class for itself? 
Wouldn't `confirmTimeout: Option[FiniteDuration]` express all that is needed?

Reply:
I can do so if you prefer.
I thought it was more clear to have `publishConfirm: Option[AmqpPublishConfirmConfiguration] = None` in the Sink settings rather than simply `confirmTimeout: Option[FiniteDuration]` since the `Option` here represent the activation of the "confirm" feature. When I read `confirmTimeout: Option[FiniteDuration]`, I first think that it's the timeout that is optional, and not the "confirm" feature itself :/

Reply:
Ok, makes sense.